com.chartboost.mediation.demo/Assets/Plugins/Android/mainTemplate.gradle

```diff
@@ -0,0 +1,137 @@
+// Android Resolver Repos Start
+([rootProject] + (rootProject.subprojects as List)).each { project ->
+    project.repositories {
+        def unityProjectPath = $/file:///**DIR_UNITYPROJECT**/$.replace("\\", "/")
+        maven {
+            url "https://maven.google.com"
+        }
+        maven {
+            url "https://cboost.jfrog.io/artifactory/helium" // Assets/Samples/Chartboost Mediation/4.0.0/Chartboost Mediation/Editor/ChartboostMediationDependencies.xml:7
```

Comment:
Small fyi: this repo will change too. But, it should be ok for now until the new one is created.